### PR TITLE
Add Korean Animal

### PR DESCRIPTION
**Pull request by swimmingkiim**
## Summary of the Pull Request
- Add Korean Animal language data
- Add Korean barrel file
- Add Language barrel file

 
## References
- [Language Rules](https://github.com/Holder-inc/random_nickname/blob/main/lib/languages/README.md)

## PR Checklist
* [x] Passed `dart format`
* [x] Passed `dart analyze --fatal-infos`
* [x] Tests added/passed

## Validation Steps Performed
- Got the word list from [official Korean dictionary](https://stdict.korean.go.kr/search/searchView.do)

### CODE REVIEW

File: lib/languages/languages.dart [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/// Barrel file for Languages barrel files
+/// Please sort alphabetically
+
+export 'kor/kor.dart';
```
Comment (KKimj) [inline lib/languages/languages.dart:4]: Would we go with Alpha-3 code?

Ref: https://www.iban.com/country-codes

Comment (swimmingkiim) [reply lib/languages/languages.dart:4]: I'm not so sure, will country codes cover all languages? 
I'm afraid that language is more comprehensive than country.

ex)
스크린샷 2022-10-07 오후 4 33 57
스크린샷 2022-10-07 오후 4 34 18

Comment (KKimj) [reply lib/languages/languages.dart:4]: We can choose like below
- KR ( only Alpha 2 code )
- KOR ( only Alpha 3 code )
- KR_KOR ( combine Alpha 2, 3 code )

kr.dart
kor.dart
kr_kor.dart

Comment (swimmingkiim) [reply lib/languages/languages.dart:4]: I don't think the suggested methods do not meet my question. 
My question was "will country codes cover all languages?"

For example, if I want to add Latin(a language that no country uses anymore), will it be able to express that in the boundary of country codes? Or a language that minorities use?

Comment (KKimj) [reply lib/languages/languages.dart:4]: Oh I got it.

We should find out another package with l10n, then just opt to follow it!

Comment (swimmingkiim) [reply lib/languages/languages.dart:4]: Here's [language list supported by Google Cloud Translation](https://cloud.google.com/translate/docs/languages)
[This one](https://github.com/michaelwittig/node-i18n-iso-countries) shows that i18n basically uses ISO-639-1 and others if necessary.

I think we should use the ISO code name, but we can still discuss what versions to choose.
What I've suggested is `639-2/T` in the `Reference` link below.

## References
- [compared list between various language code name systems](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes)

Comment (swimmingkiim) [reply lib/languages/languages.dart:4]: BTW, I've just noticed, according to [this](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes), ISO 639-2 is Alpha-3!
https://www.loc.gov/standards/iso639-2/php/code_list.php
스크린샷 2022-10-07 오후 6 44 43

Comment (KKimj) [reply lib/languages/languages.dart:4]: I like your example!

I think It will be good with ISO Standard, 639-1, 639-2 ..
You can choose whatever you want!

**Pull request merged**